spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -934,4 +938,18 @@ def expect_funnel_update_counts(user, count)
     expect(doc_auth_log.back_image_submit_count).to eq(count)
     expect(doc_auth_log.front_image_submit_count).to eq(count)
   end
+
+  def pii_like_keypaths
```

Comment:
Since this helper method is defined and used in three different spec files, I recommend consolidating and defining it in one file that stores helper methods.